### PR TITLE
AZP/TEST: Add server-side log prints

### PR DESCRIPTION
**Pull request by Alexey-Rivkin**
## What
Get server logs from ephemeral Docker containers.

## Why ?
Currently, there is no way to see stdout+stderr of the server container.

## How ?
Redirect server output to a file on the workspace volume.

### CODE REVIEW

**Review by dpressle — COMMENTED** [review]
Seems like over compilated, why not simply redirect process output to a file on the workspace volume?

File: buildlib/tools/test_mad.sh [hunk 1]
```diff
@@ -45,7 +46,11 @@ docker_run_srv() {
         -v /hpc/local:/hpc/local \
         --ulimit memlock=-1:-1 --device=/dev/infiniband/ \
         $IMAGE \
-        bash -c "${PWD}/install/bin/ucx_perftest -K ${HCA}"
+        bash -c "${PWD}/install/bin/ucx_perftest -K ${HCA} > ${PWD}/ucx_perf_srv.log 2>&1"
```
Comment (dpressle) [inline buildlib/tools/test_mad.sh:49]: since we are starting the server 2 times, we should probably generate 2 different logs for each test.

Comment (Alexey-Rivkin) [reply buildlib/tools/test_mad.sh]: Added unique names for the server log files to be on the safe side.

**Comment by Alexey-Rivkin** [comment]
> Seems like over compilated, why not simply redirect process output to a file on the workspace volume?

Good idea, thanks.

File: buildlib/tools/test_mad.sh [hunk 2]
```diff
@@ -45,7 +47,13 @@ docker_run_srv() {
         -v /hpc/local:/hpc/local \
         --ulimit memlock=-1:-1 --device=/dev/infiniband/ \
         $IMAGE \
-        bash -c "${PWD}/install/bin/ucx_perftest -K ${HCA}"
+        bash -c "${PWD}/install/bin/ucx_perftest -K ${HCA} > \
+            ${PWD}/ucx_perf_srv_${test_name}_${BUILD_BUILDID}.log 2>&1"
+}
+
+srv_logs() {
```
Comment (yosefe) [inline buildlib/tools/test_mad.sh:54]: maybe we can print the logs during server stop stage?

Comment (Alexey-Rivkin) [reply buildlib/tools/test_mad.sh:54]: Good idea. The only drawback is that you will need to know where to look for logs, rather than dedicated steps now.

Comment (yosefe) [reply buildlib/tools/test_mad.sh:54]: Add ".. and print logs" to the stage name :)

**Review by yosefe — APPROVED** [review]

**Pull request merged**